thread/src/components/Common/TransparentButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Item, Input, Text } from 'native-base';
+import styled from 'styled-components';
+
+const TermsButtonStyle = styled(Text)`
```

Comment:
Have a look at this please, https://docs.nativebase.io/Components.html#button-transparent-headref

Reply:
Rather use native base buttons